config/locales/views.fr.yml

```
@@ -322,6 +322,8 @@ fr:
         phone_number: Numéro de télephone
         your_contact_in_the_company: 'Contact de l’entreprise %{company_name} :'
       matches:
+        add_expert: Ajouter un ou une expert·e
+        add_expert_label: Rechercher par nom d'équipe, antenne, institution...
```

```suggestion
        add_expert_label: Rechercher par nom d’équipe, antenne, institution...
```
Je me suis dit, je mets un commentaire, j’en mets pas... et puis j'ai craqué 😆